tests/test_api.py

```diff
@@ -220,13 +220,20 @@ def make_request(request):
             with patch.object(MockHa, 'restart', Mock(return_value=(retval, "foo"))):
                 request = make_request('{"role": "master", "postgres_version": "9.5.2"}')
                 MockRestApiServer(RestApiHandler, request)
+            with patch.object(MockHa, 'schedule_future_restart', Mock(side_effect=Exception, return_value=retval)):
```

Comment:
I guess this test is obsolete, since we are not handling exceptions in this place anymore.
Plus the same for line#233